tests/format/typescript/typeparams/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -404,12 +404,75 @@ export type OuterType3<
 
 export type OuterType4<
   LongerLongerLongerLongerInnerType extends
-    LongerLongerLongerLongerLongerLo | ngerLongerLongerOtherType,
+    | LongerLongerLongerLongerLongerLo
+    | ngerLongerLongerOtherType,
```

Comment:
Should we keep the extends part inline?

**Prettier pr-14858**
[Playground link](https://deploy-preview-14858--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACGATwzjwHkBXebAFWLgBYAeAHSjzwBloBzObblD4De-QcICSUKPzolxY0SKGKV2KTNr086eFAAmAZzYcOAQTwAfPACEANGwB8eALx5geAIZI8ARjwAvgDcIHYgEBgwAJbQhsigntjYEADuAAqJCHEongA2KZ6EcWEARtieYADWcDAAyp4AtnCcUTLIAGZ5hnCl5VU1tRgVrTzIMNgUPSDdDVFjE1OGI7lwAIoUEPAdXVMAVoZotctrG1tInbndYQCOp3BpyRjZIJ6GALQycPpfoSDjnlFciMAMIQBoNTzIF65XK-JZCFZmGDjKIlKj3MStODbS5TAAWMAauQA6niovBDEMwHBallyVEAG7kwhQsCGYogBmTKTfWC1MDYKKRMwGWpEFY4q4gDDJbrE8oYKEyuDdbAM7FhVqqmAPTw8CGSqZDbCqqFEEiGAVCmC-GWtGDEqL6GB45AADgADGFsHBblEfbr9ZDzjswjBPCVHc7XUgAExhCjdGgR7IXKVwBolL7ffScTxCCh6uAAMRwEORIyhnioEBAAQCQA)
<!-- prettier-ignore -->
```sh
--parser typescript
```

**Input:**
<!-- prettier-ignore -->
```tsx
export type OuterType4<
  LongerLongerLongerLongerInnerTypeLongerLongerLongerLongerInnerType extends
    A | B,
> = { a: 1 };
```

**Output:**
<!-- prettier-ignore -->
```tsx
export type OuterType4<
  LongerLongerLongerLongerInnerTypeLongerLongerLongerLongerInnerType extends
    | A
    | B,
> = { a: 1 };

```

Reply:
Ideally, yes? But current `main` outputs this code, so if we address this, it should be addressed  as other issue.

**Prettier 2.8.8**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACGATwzjwHkBXebAFWLgBYAeAHSjzwBloBzObblD4De-QcICSUKPzolxY0SKGKV2KTNr086eFAAmAZzYcOAQTwAfPACEANGwB8eALx5geAIZI8ARjwAvgDcIHYgEBgwAJbQhsigntjYEADuAAqJCHEongA2KZ6EcWEARtieYADWcDAAyp4AtnCcUTLIAGZ5hnCl5VU1tRgVrTzIMNgUPagNJXD6+nOcnkIUnnwAYjgNnjDRQsggnlQQoSAAFjANuQDqZ1HwhkNgcLVZ91EAbveEB2CGxSBWt1cGlyjxth0ulMAFaGNC1Ea5OAARQoEHgkNy3TCQ2wwIORBIhjA2CikVOGFJsGuUX0MDOyAAHAAGHHJbrXcoYA6UuDAj5wU4ARzR8FBEWyh0MAFoZHM5qdsHARVElaC1hCkJ0sVNug0omMJrrEXAzLtSSUqHA0mJWoKtVCwoYTaj0fbtdiQDBPCUaXSGUgAExhcaeKK5EYAYQgDU1qEMAFZThRujQfdkPVMPpMpAtYLUSWSYGYDLUiEjMd0AgEgA)
<!-- prettier-ignore -->
```sh
--parser typescript
```

**Input:**
<!-- prettier-ignore -->
```tsx
export type OuterType4<
  LongerLongerLongerLongerInnerTypeLongerLongerLongerLongerInnerType extends
    A | B,
> = { a: 1 };
```

**Output:**
<!-- prettier-ignore -->
```tsx
export type OuterType4<
  LongerLongerLongerLongerInnerTypeLongerLongerLongerLongerInnerType extends
    | A
    | B
> = { a: 1 };

```

Reply:
But we change output of this test.(line 407 in this file) The original didn't break things after `extends`.

Reply:
I don't think that should be a concern here. Because "original" means current `main` branch and there is no change from the 2.8.8 behavior. So this is a different issue.